api/api.go

```diff
@@ -83,3 +87,26 @@ func ReadJSONBody(body io.ReadCloser, v interface{}) error {
 
 	return nil
 }
+
+func (api *API) latestAPIHandler(permissions AuthHandler) {
+	switch api.cfg.LatestVersion {
+	case "v1":
+		api.Router.HandleFunc("/jobs", api.GetJobsHandler).Methods("GET")
+		api.Router.HandleFunc("/jobs", api.CreateJobHandler).Methods("POST")
+		api.Router.HandleFunc("/jobs/{id}", api.GetJobHandler).Methods("GET")
+		api.Router.HandleFunc("/jobs/{id}", permissions.Require(update, api.PatchJobStatusHandler)).Methods("PATCH")
+		api.Router.HandleFunc("/jobs/{id}/number_of_tasks/{count}", api.PutNumTasksHandler).Methods("PUT")
+		api.Router.HandleFunc("/jobs/{id}/tasks", api.GetTasksHandler).Methods("GET")
+		api.Router.HandleFunc("/jobs/{id}/tasks", permissions.Require(update, api.CreateTaskHandler)).Methods("POST")
+		api.Router.HandleFunc("/jobs/{id}/tasks/{task_name}", api.GetTaskHandler).Methods("GET")
+	default:
+		api.Router.HandleFunc("/jobs", api.GetJobsHandler).Methods("GET")
```

Comment:
Perhaps lines 103 to 110 should be extracted into a method as they are repeating lines 94 to 101.